business_requirement_deliverable_project/models/business.py

```diff
@@ -13,12 +13,10 @@ def generate_projects_wizard(self):
             br_ids=self).generate_project_wizard()
         return res
 
-    linked_project = fields.Many2one(
-        string='Linked project',
+    project_ids = fields.One2many(
         comodel_name='project.project',
-        groups='project.group_project_user',
```

Comment:
@YogeshMahera-SerpentCS I didn't expect this change. Why is necessary remove these groups, readonly, copy, etc...?